helm/cluster-api/templates/bootstrap/certificate.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cert-manager.io/v1
```

Comment:
Currently in AWS we are using a single certificate for all CAPI providers/components (core, bootstrap, controlplane). Here we align with upstream and use a certificate per provider/component.